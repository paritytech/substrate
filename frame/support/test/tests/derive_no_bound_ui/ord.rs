trait Config {
	type C;
}

#[derive(frame_support::OrdNoBound)]
struct Foo<T: Config> {
	c: T::C,
}

fn main() {}
