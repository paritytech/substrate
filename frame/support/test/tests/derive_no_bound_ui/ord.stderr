error[E0599]: the method `cmp` exists for associated type `<T as Config>::C`, but its trait bounds were not satisfied
 --> $DIR/ord.rs:9:2
  |
9 |     c: T::C,
  |     ^ method cannot be called on `<T as Config>::C` due to unsatisfied trait bounds
  |
  = note: the following trait bounds were not satisfied:
          `<T as Config>::C: Iterator`
          which is required by `&mut <T as Config>::C: Iterator`
